packages/connect-webextension/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@trezor/connect-webextension",
-    "version": "0.0.0-beta.3",
+    "version": "9.1.12",
```

Comment:
I understand this is on purpose because we want to have the same version as `connect` and  `connect-web`.